client/src/components/generic/Header.js

```diff
@@ -58,19 +60,20 @@ export const Header = () => {
               <Menu
                 options={[
                   { label: 'Logout', onClick: clearAuthState },
-                  {
-                    label: 'Payments',
-                    onClick: () => history.push(`${Route.AdminPortalPaymentPage}`),
-                  },
-                  {
-                    label: 'Reports',
-                    onClick: () => history.push(`${Route.AdminPortalReportsPage}`),
-                  },
+                  // {
+                  //   label: 'Payments',
+                  //   onClick: () => history.push(`${Route.AdminPortalPaymentPage}`),
```

Comment:
A note for the future, don't change this now, but using javascript for navigation is bad for accessibility 
https://a11y-101.com/design/button-vs-link

Reply:
Agreed :)